specification/deviceupdate/data-plane/Microsoft.DeviceUpdate/stable/2022-10-01/deviceupdate.json

```
@@ -0,0 +1,4472 @@
+      }
+    },
+    "UpdateProviderParameter": {
+      "name": "provider",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Update provider.",
+      "x-ms-parameter-location": "method"
+    },
+    "UpdateNameParameter": {
+      "name": "name",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Update name.",
+      "x-ms-parameter-location": "method"
+    },
+    "UpdateVersionParameter": {
+      "name": "version",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Update version.",
+      "x-ms-parameter-location": "method"
+    },
+    "FileIdParameter": {
+      "name": "fileId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "File identifier.",
+      "x-ms-parameter-location": "method"
+    },
+    "IfNoneMatchParameter": {
+      "name": "If-None-Match",
+      "in": "header",
+      "required": false,
+      "description": "Defines the If-None-Match condition. The operation will be performed only if the ETag on the server does not match this value.",
+      "type": "string",
+      "x-ms-parameter-grouping": {
+        "name": "access-condition"
+      },
+      "x-ms-parameter-location": "method"
+    },
+    "OperationIdParameter": {
+      "name": "operationId",
+      "in": "path",
+      "description": "Operation identifier.",
+      "minLength": 1,
+      "maxLength": 256,
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "LogCollectionIdParameter": {
+      "name": "operationId",
+      "in": "path",
+      "description": "Log collection identifier.",
+      "minLength": 1,
+      "maxLength": 256,
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method",
+      "x-ms-client-name": "logCollectionId"
+    },
+    "DeviceIdParameter": {
+      "name": "deviceId",
+      "in": "path",
+      "description": "Device identifier in Azure IoT Hub.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "ModuleIdParameter": {
+      "name": "moduleId",
+      "in": "path",
+      "description": "Device module identifier in Azure IoT Hub.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "DeviceClassIdParameter": {
+      "name": "deviceClassId",
+      "in": "path",
+      "description": "Device class identifier.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "TagNameParameter": {
+      "name": "tagName",
+      "in": "path",
+      "description": "Tag name.",
+      "required": true,
+      "type": "string",
+      "x-ms-parameter-location": "method"
+    },
+    "GroupIdParameter": {
+      "name": "groupId",
+      "in": "path",
+      "description": "Group identity.",
```

Consistency with other field descriptions:

```suggestion
      "description": "Group identifier.",
```